README.md

```diff
@@ -1148,7 +1154,14 @@ machine twitch login my_twitch_account_name password my_twitch_password
 ```
 To activate authentication with the `.netrc` file you should pass `--netrc` to yt-dlp or place it in the [configuration file](#configuration).
 
-The default location of the .netrc file is `$HOME` (`~`). On Windows, if `$HOME` is not present, `$USERPROFILE` (generally `C:\Users\<user name>`) or `${HOMEDRIVE}${HOMEPATH}` is used
+The default location of the .netrc file is `${HOME}` (`~`).
+
+### Notes about environment variables
```

Comment:
I don't particularly like where I've put this, but cant see any better locations without making it an h1 header

Reply:
I my opinion it should be a h1 header (and be closer to the top), but it's your call

Reply:
It doesn't deserve its own main section. Most of this isnt even really yt-dlp's documentation, but general info. The only actual documentation is

> * yt-dlp also allow using UNIX-style variables on Windows for path-like options; e.g. `--output`, `--config-location`

I'm tempted to move this out to the Wiki 🤔

Reply:
> I'm tempted to move this out to the Wiki 🤔

But then people will have no idea what `${APPDATA}` is

On a side note, IMO the docs should use `%VARIABLE%` instead of `${VARIABLE}`, but it's your call

Reply:
We have a good idea to navigate them to. It's called Google

Reply:
> On a side note, IMO the docs should use %VARIABLE% instead of ${VARIABLE}, but it's your call

https://github.com/yt-dlp/yt-dlp/pull/4906

In the end, I decided on $ since yt-dlp will resolve it in both OSes, atleast for the relevent options

Reply:
I disagree, because Windows Explorer (which most users will be using to find stuff) won't resolve ${VARIABLE}. That said, it's your call